include/eve/detail/function/simd/x86/friends.hpp

```diff
@@ -149,6 +150,15 @@ namespace eve::detail
         else  if constexpr( c == category::uint32x4 )             return _mm_cmpeq_epi32(v,w);
         else  if constexpr( c == category::uint16x8 )             return _mm_cmpeq_epi16(v,w);
         else  if constexpr( c == category::uint8x16 )             return _mm_cmpeq_epi8 (v,w);
+        else  if constexpr( c == category::int64x4 || c == category::uint64x4 ||
+                            c == category::int32x8 || c == category::uint32x8
+                          )
+        {
+          using f_t = as_floating_point_t<wide<T, N, ABI>>;
+          auto fv   = bit_cast( v , as_<f_t>{});
+               fv  ^= bit_cast( w, as_<f_t>{});
+          return bit_cast ( !to_logical( fv ), as_<logical<wide<T, N, ABI>>>{} );
```

Comment:
This is 4 instructions, right?

xor + not + zero + andnot ?

(well zero maybe hoisted out).